SpatialGDK/Source/SpatialGDK/Private/Interop/SkeletonEntities.cpp

```diff
@@ -1,26 +1,36 @@
 #include "Interop/SkeletonEntities.h"
```

Comment:
Scary looking diff, look at the file instead. This is emptying it save for a couple of utility functions.
The other features have been dispatched in the next commits.